modules/rpc/thriftrw-plugin-thriftsync/README.md

```diff
@@ -3,17 +3,44 @@
  *.thrift file. With the use of thriftsync plugin, a user who needs to build a service should be able
```

Comment:
- of thriftsync plugin --> of the thriftsync plugin
- a user who needs to build --> a user building